build-python-package/action.yml

```
@@ -9,6 +9,10 @@ inputs:
     description: Obsolete; there is no longer a difference between dry-run and real build
     required: false
     default: false
+  optional-deps:
+    description: Specify optional dependencies to install e.g, `foo,bar`.
+    default: 'all'
```

See https://github.com/cylc/release-actions/pull/127/files#r2363045535
```suggestion
    description: Specify optional dependencies to install e.g, `[foo,bar]`.
    default: '[all]'
```